package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "isolated-externals-plugin",
+  "version": "0.0.1",
```

Comment:
Since we know `semantic-release` will always do an initial release of `1.0.0` let's spare the initial confusion and set this to `1.0.0`.